test/handlers/FranchiserFactoryHandler.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {Test, console2} from "forge-std/Test.sol";
+
+import {IVotingToken} from "src/interfaces/IVotingToken.sol";
+import {VotingTokenConcrete} from "test/VotingTokenConcrete.sol";
+import {FranchiserFactory} from "src/FranchiserFactory.sol";
+import {Franchiser} from "src/Franchiser.sol";
+
+contract FranchiserFactoryHandler is Test {
+    FranchiserFactory public factory;
+    Franchiser public franchiser;
+
+    constructor(FranchiserFactory _factory) {
+        factory = _factory;
+        franchiser = new Franchiser(IVotingToken(address(factory.votingToken())));
+    }
+
+    function validAddress(address _address) internal view returns (bool valid) {
+        valid =
+            (_address != address(0)) && (_address != address(factory.votingToken()) && (_address != address(factory)));
+    }
+
+    function handler_fund(address _delegator, address _delegatee, uint256 _amount) external {
+        vm.assume(_delegator != _delegatee);
+        vm.assume(validAddress(_delegator));
+        vm.assume(validAddress(_delegatee));
+        _amount = bound(_amount, 0, 100000);
```

Comment:
I think we should probably have a higher upper bound here

Reply:
And same in the other function. It may be worth making this an internal function as it is also used in the recall function.

Reply:
Done: https://github.com/uniswapfoundation/franchiser/pull/1/commits/82a76a48cb84524c473ed406a239bec67dfefe2d

Reply:
Sorry I should of been more specific. Maybe lets makes this 100 milion e18. That seems reasonably high for what Uniswap would deposit.

Reply:
Done: https://github.com/uniswapfoundation/franchiser/pull/1/commits/bfcacbfc239cb07ad865668e58087030a2b894b8